.github/workflows/push-build.yaml

```
@@ -0,0 +1,31 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+    tags:
+      - v*
+jobs:
+  push-master-image-on-docker:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: 18.04
```

Let's stay consistent with what we have today